app/services/enqueue_index_jobs.rb

```diff
@@ -0,0 +1,70 @@
+# Indexes a book version thru the SQS queuing system
+#
+# @params indexing_version e.g., Search::BookVersions::I1::IndexingStrategy::VERSION
+class EnqueueIndexJobs
+  def initialize
+    @todo_jobs_queue = TodoJobsQueue.new
+  end
+
+  def process_rex_releases
+    rex_releases = OpenStax::RexReleases.new
+    books_ids = rex_releases.map{ |release| release.books }.flatten
+
+    books_ids.each do |book_id|
+      ACTIVE_INDEXING_STRATEGIES.each do |indexing_strategy|
+        process_book(book_id, indexing_strategy.new.version)
+      end
+    end
+
+    check_for_books_to_delete
+
+    AutoScaling.set_desired_capacity(group_name: asg_name, desired_capacity: @todo_jobs_queue.count)
```

Comment:
In that PR I changed this to just an ASG instance (more instance methods, less class methods)